includes/class-wc-gateway-ppec-cart-handler.php

```diff
@@ -301,7 +301,7 @@ public function enqueue_scripts() {
 
 		wp_enqueue_style( 'wc-gateway-ppec-frontend-cart', wc_gateway_ppec()->plugin_url . 'assets/css/wc-gateway-ppec-frontend-cart.css' );
 
-		$is_cart     = is_cart() && 'yes' === $settings->cart_checkout_enabled;
+		$is_cart     = is_cart() && ! WC()->cart->is_empty() && 'yes' === $settings->cart_checkout_enabled;
```

Comment:
Same note on previous PR, not sure if there's a case where the cart may not be populated.

Reply:
Doesn't seem so, as it's used in other places, and is initialized on every front-end request: https://github.com/woocommerce/woocommerce/blob/7b56840f360158b3f4a4f4ff924339fd70b82c17/includes/class-woocommerce.php#L479